pkg/web/static/openapi.json

```diff
@@ -588,6 +588,9 @@
           },
           "404": {
             "description": "Transaction not found"
+          },
+          "501": {
+            "description": "Not implemented" 
```

Comment:
This endpoint hasn't been implemented yet, so I wanted to reflect that in the documentation. If you'd prefer that we remove it completely until it is implemented, that will be fine.